packages/components/src/RadioGroup.test.tsx

```diff
@@ -1,26 +1,22 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import RadioGroup from './RadioGroup';
 import RadioItem from './RadioItem';
 
 it('shows no markup with no children', () => {
-  const radio = mount(<RadioGroup onChange={jest.fn()} />);
-  expect(radio.html()).toBe(null);
-  radio.unmount();
+  const radio = render(<RadioGroup onChange={jest.fn()} />);
+  expect(radio.baseElement.firstChild.firstChild).toBe(null);
```

Comment:
Prefer to get it from `container` instead of base element. Can check innerHtml or childElementCount (or both):
```suggestion
  const { container } = render(<RadioGroup onChange={jest.fn()} />);
  expect(container.childElementCount).toBe(0);
  expect(container.innerHtml).toBe('')
```